charts/camunda-platform/Chart.yaml

```diff
@@ -22,6 +22,13 @@ dependencies:
   - name: identity
     version: 8.1.1
     condition: "identity.enabled"
+    # NOTE: This is used to share Identity image details with its subchart Keycloak.
+    #       It should be part of Identity but Helm 3 missing this option currently (v3.10.x).
+    # TODO: Move this to Identity subchart once "export-values" is implemented.
+    #       https://github.com/helm/helm/pull/10804
```

Comment:
These parent-child access issues is so annoying

Reply:
super annoying tbh 😂️